src/v2/components/MessageList/MessageList.tsx

```diff
@@ -540,9 +531,6 @@ const MessageListWithContext = <
     if (!channel?.state.isUpToDate) {
       await reloadChannel();
     }
-    if (flatListRef.current) {
-      flatListRef.current.scrollToIndex({ index: 0 });
```

Comment:
This call always fails, probably because flatlist is not loaded at this point. But overall UX pretty much works even without this function call. So just gonna remove it.

```
scrollToIndex failed ...  max allowed index is -1.
```